gtk/src/toga_gtk/libs/gtk.py

```diff
@@ -6,6 +6,41 @@
 gi.require_version("Gdk", gtk_version)
 gi.require_version("Gtk", gtk_version)
 
+# no-covering because we cannot test all DE and TOGA_GTKLIB variables in CI
+# Detect the GTK DE-specific library to be used.  The TOGA_GTKLIB value will
+# be prioritized, or, if no GTK4 library is specified, falls back to detecting
+# with desktop environment.
+if gtk_version == "4.0":  # pragma: no cover
+    if os.getenv("TOGA_GTKLIB") == "Adw":
+        gtklib = "Adw"
+    elif os.getenv("TOGA_GTKLIB") == "None":
+        gtklib = None
+    elif os.getenv("TOGA_GTKLIB") in {None, ""}:
+        # No TOGA_GTKLIB specified; autodetect from DE
+        if "GNOME" in (os.getenv("XDG_CURRENT_DESKTOP") or "").split(":"):
```

Comment:
`os.getenv` has a built-in defaulting capability.
```suggestion
        if "GNOME" in (os.getenv("XDG_CURRENT_DESKTOP", "").split(":"):
```